src/MovieCard/MovieCard.js

```diff
@@ -53,13 +53,24 @@ class MovieCard extends Component {
       }
   }
 
+  addFavorited = async(event) => {
+    const movieIdToUpdate = event.target.id;
+    console.log(movieIdToUpdate);
+  }
```

Comment:
I've heard present tense verbs are usually best convention for function names. Maybe something to consider for this addFavorited method